### DISCUSSION TITLE
Android Source Code

### DISCUSSION THREAD

**Discussion by zhouzhuli** [yasirkula/UnityAndroidRuntimePermissions#13] [General]
Thanks for the author's work, this plugin is very useful and solved my problem very well. I hope that the author can also upload the source code of the Android side to Git, so that if there is any problem later, we can modify and maintain it ourselves.
Thanks again!

**Comment by yasirkula** [comment]
Please see: https://github.com/yasirkula/UnityAndroidRuntimePermissions/tree/master/.github/AAR%20Source%20(Android) 😃
